runnables/binning.py

```diff
@@ -12,7 +13,9 @@ def build_argument_parser():
     parser.add_argument("--columns", type=str, required=True)
     parser.add_argument("--bin_methods", type=str, required=False)
     parser.add_argument("--bin_nums", type=str, required=False)
+    parser.add_argument("--bin_input_table", type=str, required=False)
     parser.add_argument("--reverse_cumsum", type=bool, default=False)
+    parser.add_argument("--two_dim_bin_cols", type=str, required=False)
```

Comment:
Can we distinguish two number of dims using `--columns` only?

Reply:
--columns can specify any number of columns to calcluate the basic statistical result and bin probability.
And inside these columns, we can pick two columns to calculate the two dimension binning probability.